pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -181,7 +181,7 @@ def cache_training_step_metrics(self, opt_closure_result):
         self.logged_metrics.update(logged_metrics_tmp)
         self.cached_results.legacy_batch_log_metrics.update(logged_metrics_tmp)
 
-    def log_metrics(self, metrics, grad_norm_dic, step=None):
+    def log_metrics(self, metrics, grad_norm_dic, step=None, log_train_step_metrics=False):
```

Comment:
could you add description of the new arg under docstring?

Reply:
done !